test/Test/Password.hs

```diff
@@ -11,14 +11,11 @@ import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
 
 test_pwdHashVerified :: [TestTree]
-test_pwdHashVerified = pure $ testProperty "Password verification" pwdProperty
-
-pwdProperty :: Property
-pwdProperty = property $ do
-  randomPwd <- forAll genPwd
-  whenRightM (runExceptT $ mkPasswordHashWithPolicy BC.fastBcryptHashingPolicy randomPwd) $ \pwdHash ->
-    assert $ verifyPassword randomPwd pwdHash
-
+test_pwdHashVerified = pure $ testProperty "Password verification" $
```

Comment:
Could you move `$ property $ do` on this line and align code with 4 spaces? Just to make code more consistent with `JwtSpec`